### ISSUE TITLE
Support relative path in dependency

### ISSUE THREAD

**Issue by WiSaGaN** [kbknapp/cargo-outdated#96]
This is related to #50.
If we create two local cargo projects under the same working directory `foo` and `bar` by running `cargo new foo` and `cargo new bar`, and having `bar` depending on `foo` by including the following in the `Cargo.toml` of `bar`:
```
[dependencies.foo]
version = "0.1"
path = "../foo"
```
It will compile successfully using `cargo build`. `cargo outdated` will throw an error:
```
cargo outdated
error: failed to load source for a dependency on `foo`

Caused by:
  Unable to update file:///tmp/foo

Caused by:
  failed to read `/tmp/foo/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
```
I am using `cargo-outdated 0.6.1`.

**Comment by Frederick888** [comment]
Should be fixed by #97.

The `replace_path_with_absolute()` should be applied to both compatible and latest manifest files. This was done by calling `write_manifest_semver_with_paths()` right after the files were copied to temporary workspaces but later `write_manifest_semver_with_paths()` was removed in [`v0.6.0`](https://github.com/kbknapp/cargo-outdated/pull/87/files#diff-eb3c13e0392e1efb496ff86d829d78caL194) and I forgot it in the latest temporary workspace.

Could you please try the PR out?

**Comment by WiSaGaN** [comment]
Yes, the PR fixes the problem on my local machine.
```
All dependencies are up to date, yay!
```
Thanks!

**Comment by Frederick888** [comment]
Published as v0.6.2
